source/common/config/grpc_subscription_impl.h

```diff
@@ -81,6 +81,8 @@ class GrpcSubscriptionImpl : public Config::Subscription<ResourceType>,
     sendDiscoveryRequest();
   }
 
+  const std::string& versionInfo() const override { return request_.version_info(); }
```

Comment:
This, and the below in http_subscription_impl.h cause a proto string compatibility issue for us because our protobuf generated code returns something which is convertible into an std::string rather than an std::string resulting in this trying to take a reference to a temporary. Will push a PR to address this shortly.